tools/docker/docker-artifacts/newrelic/newrelic.yml

```diff
@@ -197,10 +197,14 @@ common: &default_settings
     # by providing a comma separated list of status codes.
     # The default is to exclude 404s. If you want to override
     # this, you must provide any new value as an empty list is ignored.
-    ignore_status_codes: 404
+    ignore_status_codes: [
+      404, // resource doesn't exist
+      405, // invalid request type
+      422  // validation error
```

Comment:
@kencheeto should i pre-emptively add the other ignored status codes from submissions-api, or wait till/if we encounter them? probably wait?

400, // BadRequestError -- missing required query parameter or extra invalid query parameter
      401, // UnauthorizedError -- Invalid/missing JWT or user doesn't have necessary AuthZ privileges
      403, // Forbidden (we don't actually use this yet, but might)
      406, // NotAcceptableError -- Accept headers asking for format our API doesn't support

Reply:
I would think adding `401` would be good because our application will soon support JWTs.  The other ones shouldn't be too bad to add either.

Reply:
OK, will do!